docker/entrypoint.sh

```diff
@@ -1,24 +1,108 @@
 #!/bin/bash
+
+show_progress() {
+    message="$1"
+    current="$2"
+    total="$3"
+    percent="$4"
+
+    done=$(bc <<< "scale=0; 40 * ${percent%.*} / 100" )
+    todo=$(bc <<< "scale=0; 40 - $done" )
+    done_sub_bar=$(printf "%${done}s" | tr " " "#")
+    todo_sub_bar=$(printf "%${todo}s" | tr " " "-")
+    echo -e "$message $current/$total [${done_sub_bar}${todo_sub_bar}] ${percent}%"
+}
+
+node_verification() {
+     REGEX="^.?\[.*\].?\[.*\].?(Validating|Validated) chunk no\. ([0-9]+) out of ([0-9]+)\. Progress: ([0-9]+\.[0-9][0-9])\%.*$"
+     echo -e "Node verification..."
+     rm -f $CARDANO_NODE_SOCKET_PATH
+     while [ ! -S "$CARDANO_NODE_SOCKET_PATH" ] ; do
+          line=$(tail -n 1 /logs/node.log)
+          if [[ "$line" =~ $REGEX ]]; then
+               current="${BASH_REMATCH[2]}"
+               total="${BASH_REMATCH[3]}"
+               percent="${BASH_REMATCH[4]}"
+
+               show_progress "Node verification: Chunk" $current $total $percent
+          fi
+          sleep 1
+    done
+    echo -e "Node verification: DONE$(printf "%100s")"
+}
+
+node_synchronization() {
+    echo -e "Node synchronization..."
+    epoch_length=$(jq -r .epochLength $GENESIS_SHELLEY_PATH)
```

Comment:
This code was taken from here:
https://forum.cardano.org/t/check-sync-status-of-node
Please check it carefully